CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.0.2] - 2018-12-13
+* Updates for extension to work with new router. Removed usage of subscription stream and instead update based on current pathname.
```

Comment:
```suggestion
* Updates for extension to work with PWA 6.1. Removed usage of subscription stream and instead update based on current pathname.
```
externals don't know the term "new router"

Reply:
Changed wording